config/samples/simplenificluster.yaml

```
@@ -5,13 +5,31 @@ metadata:
 spec:
   service:
     headlessEnabled: true
-  zkAddress: "zookeeper.zookeeeper:2181"
-  zkPath: "/simplenifi"
-  clusterImage: "apache/nifi:1.15.3"
-  oneNifiNodePerNode: false
+    labels:
+      cluster-name: simplenifi
+  zkAddress: "zookeeper.zookeeper:2181"
+  zkPath: /simplenifi
+  externalServices:
+    - metadata:
+        labels:
+          cluster-name: driver-simplenifi
+      name: driver-ip
+      spec:
+        portConfigs:
+          - internalListenerName: http
+            port: 8080
+        type: LoadBalancer
+  clusterImage: 'apache/nifi:1.12.1'
```

```suggestion
  clusterImage: "apache/nifi:1.15.3"
```
The latest version tested